jstests/aggregation/bugs/server33149.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    "use strict";
+
+    // For arrayEq.
+    load("jstests/aggregation/extras/utils.js");
+
+    let indexDB = db.getSiblingDB("create_indexess_db");
+    assert.commandWorked(indexDB.dropDatabase());
+
+    // Create an index with key: {x : 1}, name : "x_1
+    assert.commandWorked(indexDB.runCommand({ createIndexes : "test", indexes : [ { key : { x : 1 }, name : "x_1" } ] }));
```

Comment:
All of the code in our repository is formatted by clang-format, including our javascript tests. It looks like this code needs to be formatted by clang-format. Please see here for a guide of that: https://github.com/mongodb/mongo/wiki/Server-Code-Style#clang-format

This applies to the C++ code as well - thanks.